nowcasting_dataset/config/model.py

```diff
@@ -240,6 +255,13 @@ class InputData(BaseModel):
         ),
     )
 
+    start_datetime: datetime = Field(
```

Comment:
Is this needed, given that the `StartEndDatetimeMixin` will inject a `start_datetime` and `end_datetime` fields?

Reply:
I was thinking that these are the general or default ones, and then in PV and GSP, separate ones can be set

Reply:
Actually, that raises an interesting question:

What's the use-case where we'd want differente start_datetimes and end_datetimes for each datasource?  Or should start_datetime and end_datetime always be "global"

Reply:
Perhaps if one data source is dodge before a certain date, but we still want data from before this date for the batches, but with this data source missing. Perhaps this is a bit far fetched ...

Reply:
That's a good point about avoiding dodgy data...

But `Manager` starts by computing the intersection of all the t0_datetimes available across all the DataSources.  So if one data source starts really late, then all the data sources will start really late.

so, yeah, I'm starting to wonder if maybe we should keep things simple and just have a single, global `start_datetime` and `end_datetime`?  What do you think?

Reply:
Maybe its simplier to remove this bit - simplier is often better

Reply:
Yea two different ways to do it
1. Currently its just a mxinin in PV and GSP
2. other option is to have a global one which is move into PV GSP through a validation method. 

I think 1 is simple pydantic model. Can always change this in the future if we find we are changing things.